lib/ProductOpener/Display.pm

```diff
@@ -3033,81 +3033,70 @@ sub display_tag ($request_ref) {
 
 	my $title;
 
-	my $tagtype = $request_ref->{tagtype};
-	my $tagid = $request_ref->{tagid};
-	my $display_tag;
-	my $newtagid;
-	my $newtagidpath;
-	my $canon_tagid = undef;
+	local $log->context->{tags} = $request_ref->{tags};
 
-	local $log->context->{tagtype} = $tagtype;
-	local $log->context->{tagid} = $tagid;
+	init_tags_texts() unless %tags_texts;
 
-	my $tagtype2 = $request_ref->{tagtype2};
-	my $tagid2 = $request_ref->{tagid2};
-	my $display_tag2;
-	my $newtagid2;
-	my $newtagid2path;
-	my $canon_tagid2 = undef;
+	$request_ref->{current_link} = '';
+	$request_ref->{world_current_link} = '';
 
-	local $log->context->{tagtype2} = $tagtype2;
-	local $log->context->{tagid2} = $tagid2;
+	my $header_meta_noindex = 0;    # Will be set if one of the tags is related to a user
+	my $redirect_to_canonical_url = 0;    # Will be set if one of the tags is not canonical
 
-	init_tags_texts() unless %tags_texts;
+	# Go through the tags filters from the request
 
-	# Add a meta robot noindex for pages related to users
-	if (
-		(
-				(defined $tagtype)
-			and ($tagtype =~ /^(users|correctors|editors|informers|correctors|photographers|checkers)$/)
-		)
-		or (    (defined $tagtype2)
-			and ($tagtype2 =~ /^(users|correctors|editors|informers|correctors|photographers|checkers)$/))
-		)
-	{
+	foreach my $tag_ref (@{$request_ref->{tags}}) {
 
-		$header .= '<meta name="robots" content="noindex">' . "\n";
-
-	}
+		my $tagid = $tag_ref->{tagid};
+		my $tagtype = $tag_ref->{tagtype};
+		my $tag_prefix = $tag_ref->{tag_prefix};
```

Comment:
can you add a comment on what is tag_prefix ?

Reply:
sure